lisa/features/gpu.py

```diff
@@ -122,8 +122,7 @@ def install_compute_sdk(self, version: str = "") -> None:
         self._install_gpu_dep()
         try:
             # install LIS driver if required and not already installed.
-            if LisDriver.can_install:
```

Comment:
The `can_install` is called by LISA, it doesn't need to be called explicitly.

Reply:
hmm, yes, let me retry

Reply:
updated